src/compile/forward.rs

```diff
@@ -118,10 +123,15 @@ fn extract_code(
                     settings,
                     document.newline(),
                 )?;
-                println!("  Writing file {}", file_path.display());
-                fs::create_dir_all(file_path.parent().unwrap())?;
-                let mut code_file = File::create(file_path)?;
-                write!(code_file, "{}", code)?;
+
+                if files::file_differs(&file_path, &code)? {
+                    println!("  Writing file {}", file_path.display());
+                    fs::create_dir_all(file_path.parent().unwrap())?;
+                    let mut code_file = File::create(file_path)?;
```

Comment:
This looks a lot like `fs::write`?

Reply:
Cool, didn't know that... There is also a third occurrence...